medusa/providers/torrent/html/limetorrents.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
             'update': urljoin(self.url, '/post/updatestats.php'),
             'search': urljoin(self.url, '/search/tv/{query}/'),
             'rss': urljoin(self.url, '/browse-torrents/TV-shows/date/{page}/'),
+            'tv': urljoin(self.url, '/browse-torrents/TV-shows/'),
```

Comment:
`rss` here was reffering to it beeing an url for daily, just that. It isn't an xml rss or anything like that. I used `tv` becouse it is the page of latest tv shows.